terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1830,9 +1830,10 @@ def qq(*words):
                                 'Requires=docker.service gitlab-dind.service',
                                 '[Service]',
                                 # We explicitly configure Docker (see /etc/docker/daemon.json) to log to
-                                # journald, so we don't need systemd to capture process output.
-                                'StandardOutput=null',
-                                'StandardError=null',
+                                # journald, which works for `docker run` but not `docker exec`, so for
+                                # these logs we let systemd control where the process output gets logged.
```

Comment:
This sounds misleading. We specify here where standard output and error go to. So journal is not in control. We are.